src/vuestic-theme/vuestic-components/va-modal/VaModal.vue

```diff
@@ -199,43 +211,62 @@ export default {
     &:last-of-type {
       background-color: rgba(0, 0, 0, 0.6);
     }
-
-    &-enter,
-    &-leave-to {
-      opacity: 0;
+    &.transition-off {
+      opacity: 1;
     }
+    &__transition {
+      &-enter,
+      &-leave-to {
+        opacity: 0;
+        &:nth-of-type(n+3){
```

Comment:
Wondering what this does.

Reply:
it prevent blinks if nested modal was opened